gef-ui/templates/includes/application-preview/facilities.njk

```diff
@@ -23,10 +23,18 @@
 
 {% if facilities.data.length > 0 %}
   {% for item in facilities.data %}
-    <header class="is-flex is-justify-content-space-between">
-      <h3 class="govuk-heading-m" data-cy="facility-type-heading">
+    <header class="is-flex">
+      <h3 class="govuk-heading-m govuk-grid-column-one-half govuk-!-padding-left-0" data-cy="facility-type-heading">
```

Comment:
What does this change do?

Reply:
Do we need to update `application-details/facilities.njk` as well?

If so, seems that the `is-justify-content-space-between` class can be removed

Reply:
No - just the `application-preview` needs to be updated - the new designs are only for 'Acknowledged' deals which only result in the `application-preview` pages being shown

Reply:
Sounds good :)